### ISSUE TITLE
Keys not working

### ISSUE THREAD

**Issue by txj-xyz** [txj-xyz/rs3-ability-tracker#15]
`'` and `tilde(~)` do not work currently

**Comment by txj-xyz** [comment]
![](https://l.txj-dev.xyz/1Gwa7)

**Comment by txj-xyz** [comment]
![](https://l.txj-dev.xyz/oPCZR)

**Comment by txj-xyz** [comment]
additional stuff
AHK remaps with modifiers don't work
for example:

AHK code:
```
Alt & 2::
    Send {l down}
    Send {k down}
    Sleep 30
    Send {l up}
    Send {k up}
    global skip_check := 1
return
```

Where pressing ALT + 2 send l and k
and the game accepts the input of ALT + l and ALT + k.

**Comment by txj-xyz** [comment]
AutoHotKey program will not be supported, the code below sends `CTRL` instead of `ALT` within a window of 2ms, therefore I will choose not to support it due to Windows interpreting it as `CTRL + L --> ALT + L` at the same time

```ahk
Alt & 2::
    Send {l down}
    Send {k down}
    Sleep 30
    Send {l up}
    Send {k up}
    global skip_check := 1
return```

**Comment by txj-xyz** [comment]
> `'` and `tilde(~)` do not work currently

Fixed in 308af4742f00e20dd1e71b82bd47b2b193943a4b
